understanding/21/reflow.html

```diff
@@ -12,68 +12,235 @@ <h1>Understanding Reflow</h1>
-      <p>Interfaces which provide toolbars to edit content need to show both the content and the toolbar in the viewport. Depending on the number of toolbar buttons, the toolbar may need to scroll in the direction of text.</p>
+        <figure>
+          <img src="img/npr-reflow.png" width="500" alt="The npr.org site as rendered in a 320px wide viewport. The navigation in collapsed into a hamburger menu, so the global header visually is decluttered - redering only the hamburger button, npr logo an donate button by defaut. The media player has become a persistent sticky widget at the bottom of the viewport. News stories have been adjusted to fit into a single column - as only one news story lead-in can fit at a time into the reduced viewport size.">
+          <figcaption>
+            <p>The npr.org website layout reflows to accommodate different viewport sizes, which can occur due to browser resize, or use of browser zoom features. As the layout adjusts, the sections of content of the web page reflow. The distinct sections of content vertically stack, allowing for scrolling in a single direction to read. The primary navigation is initially hidden behind a disclosure widget button (a.k.a., a hamburger menu button), and the media player has been visually moved and simplified to allow it to remain persistent, but not obscure the content of the page.</p>
+          </figcaption>
+        </figure>
+
+        <p>While this 'stacking into a single column' approach may work for many web pages whose primary objective is to present sections of content for people to read (e.g., text and/or standard media such as graphics, videos that represent articles, blog posts, discussion threads, topic teasers, etc.); there are many instances of common, but complex, widgets and layouts which would become more difficult to visually understand or interact with if they were adjusted to fit in a single column layout.</p>
+        
+        <p>Thus, while following responsive web design best practices will help many websites meet Reflow, the success criterion does not mandate fully responsive web pages. There will be instances in which more complex web interfaces need to expend additional effort by instead providing revised presentation, functionality, or both when content is enlarged - and thus presented in smaller viewport sizes. In other cases, aspects of the application's content will meet one of Reflow's exceptions and may be presented in a bi-directional scrolling container within the otherwise single-direction scrolling interface.</p>
```

Comment:
I found this sentence hard to parse.  But I think maybe "and thus presented in smaller viewport sizes" can be dropped?
> There will be instances in which more complex web interfaces need to expend additional effort by instead providing revised presentation, functionality, or both when content is enlarged - and thus presented in smaller viewport sizes.

Reply:
>I found this sentence hard to parse.

seeing it out of context, i can't blame you.  what the heck was i thinking?  thank you!